csrc/scheduler/matmul.cpp

```diff
@@ -350,50 +377,85 @@ void prologSwizzle(TensorView* shared_mem_tv, const MatmulParams& params) {
      *   7|          |          |
      *    +----------+----------+
      */
+    // The first para in std::gcd is from the above derivation.
+    // The second para is from the fact that we need to split tile_size_y by
+    // n_cols and then by swizzle_period. If swizzle_period is smaller than the
+    // first para, then the bank conflict can't be fully removed.
```

Comment:
> then the bank conflict can't be fully removed.

Can you open an issue for that? We need to look deep into this.

Reply:
The statement is no longer valid. Corrected code comments.